Runtime/Scripts/MediaStreamTrack.cs

```diff
@@ -229,7 +111,15 @@ public RTCRtpReceiver Receiver
         internal RTCTrackEvent(IntPtr ptrTransceiver, RTCPeerConnection peer)
         {
             IntPtr ptrTrack = NativeMethods.TransceiverGetTrack(ptrTransceiver);
-            Track = WebRTC.FindOrCreate(ptrTrack, ptr => new MediaStreamTrack(ptr));
```

Comment:
Please create a method to avoid similar implementations.

```csharp
Track = WebRTC.FindOrCreate(ptrTrack, ptr => CreateMediaStreamTrack);

MediaStreamTrack CreateMediaStreamTrack(IntPtr ptr)
{
    ...
}
 ```

Reply:
I create method on https://github.com/Unity-Technologies/com.unity.webrtc/pull/181/commits/56c0fb94cce5801d2eeabc08c52e578716c88115